h2o-persist-s3/src/test/java/water/persist/PersistS3Test.java

```diff
@@ -34,17 +38,67 @@
   }
   @Test
   public void testS3Import() throws Exception {
+    Key k = null, k2 = null;
```

Comment:
This test remained the same, I only took the opportunity to do explicit cleanup.